lib/Utility.js

```diff
@@ -223,12 +223,14 @@ var Utility = module.exports = {
 
     //pm2 install username/module
     else if(canonic_module_name.indexOf('/') !== -1) {
-      canonic_module_name = canonic_module_name.split('/')[1];
+      if (!canonic_module_name.startsWith("@")){
```

Comment:
Tests on 0.12 failed cause startsWidth is not supported :(
I think we should write test just like this : 
`if (canonic_module_name[0] !== "@"){`